src/components/OfflineTooltip.js

```diff
@@ -17,25 +17,25 @@ const OfflineTooltip = ({
     const notAllowed = disabled || (disabledWhenOffline && offline)
 
     return (
-        <Tooltip
-            content={content || i18n.t('Not available offline')}
-            openDelay={200}
-            closeDelay={100}
-        >
-            {({ onMouseOver, onMouseOut, ref }) => (
-                <span
-                    className={cx(
-                        classes.span,
-                        notAllowed && classes.notAllowed
-                    )}
-                    onMouseOver={() => notAllowed && onMouseOver()}
-                    onMouseOut={() => notAllowed && onMouseOut()}
-                    ref={ref}
-                >
-                    {children}
-                </span>
-            )}
-        </Tooltip>
+        <>
+            <Tooltip
+                content={content || i18n.t('Not available offline')}
+                openDelay={200}
+                closeDelay={100}
+            >
+                {({ onMouseOver, onMouseOut, ref }) => (
+                    <span
+                        className={cx('tooltip', notAllowed && 'notAllowed')}
```

Comment:
I've been informed by platform team that the preferred way to do this is: 
```
 <div
          className={cx(classes.iconWrapper, {
              [classes.activeIcon]: isColumns(cols),
          })}
      >
```
https://github.com/dhis2/dashboard-app/blob/master/src/pages/edit/LayoutModal.js#L90

Reply:
Or maybe even this from ui:
https://github.com/dhis2/ui/blob/master/components/tag/src/tag.js#L22

Reply:
I used the `css` syntax as it supports things like `.tooltip > :global(button:disabled)`. Is it possible to do this with the syntax you're proposing?

Reply:
I tested it out and it works: `className={cx('tooltip', {notAllowed})}`